src/app/main/component/user/components/profile/profile.component.ts

```diff
@@ -25,10 +26,12 @@ export class ProfileComponent implements OnInit, OnDestroy {
     private announcer: LiveAnnouncer,
     private localStorageService: LocalStorageService,
     private translate: TranslateService,
-    private profileService: ProfileService
+    private profileService: ProfileService,
```

Comment:
[Member 'profileService: ProfileService' is never reassigned; mark it as `readonly`.](https://sonarcloud.io/project/issues?pullRequest=3407&open=AZKVdKI_hY9BiVF5AhU3&id=ita-social-projects-green-city-client)